make_site.py

```diff
@@ -339,15 +344,19 @@ def download_N_theorems(kind: NTheorems) -> dict:
     if DOWNLOAD:
         download(f'https://leanprover-community.github.io/mathlib4_docs/{fname}', DATA/fname)
         with (DATA/fname).open('r', encoding='utf-8') as h_file:
-            n_theorems = [Type(thm,**content) for (thm,content) in yaml.safe_load(h_file).items()]
+            n_theorems = [Type(thm, **content) for (thm, content) in yaml.safe_load(h_file).items()]
             theorems = []
             for h in n_theorems:
                 assert not (h.decl and h.decls)
+                assert not h.statement and (h.decl or h.decls)
+                statement_formalised = False
```

Comment:
I don't mind whether we go with `s` or `z`, but we'd should keep the spelling consistent with `proof_formalized`.
```suggestion
                statement_formalized = False
```

Reply:
Right. Changed everything to `z`.